README.md

```diff
@@ -81,39 +81,28 @@ In order to run competition matches you'll need to:
 
    Then close webots.
 
-3. To put the competitors' code into the right places for each of the corner zones, run
+3. Run the match:
 
     ```bash
-    competition-simulator/script/prepare-comp-match <directory containing team code> <match number> <Zone 0 TLA> <Zone 1 TLA> <Zone 2 TLA> <Zone 3 TLA>
+    competition-simulator/script/run-comp-match <directory containing team code> <match number> <Zone 0 TLA> <Zone 1 TLA> <Zone 2 TLA> <Zone 3 TLA>
     ```
 
-    Using a dash instead of a TLA if a robot is not present. This provides filenames and sets up the directory structure:
+    Using a dash instead of a TLA if a robot is not present.
```

Comment:
```suggestion
    Note: Use a dash instead of a TLA if a robot is not present.
```

Reply:
As this part isn't new in this PR I'm going to leave this. I agree that that would be a better sentence though, so feel free to push it on `master` (ideally after this PR merges to avoid conflicts).